### ISSUE TITLE
Issues replicating biomedparse dice score

### ISSUE THREAD

**Issue by researchenjoyer1652** [microsoft/BiomedParse#117]
Hi BiomedParse team,

Thank you for releasing BiomedParse v2 and the 3D inference code. I am trying to evaluate the pretrained v2 checkpoint on BraTS 2021 glioma cases without fine-tuning, and I wanted to ask whether I may be using the model/prompts/preprocessing incorrectly. The reason is because I'm unfortunately unable to replicate the dice scores that were reported in the paper. The highest dice score I've obtained across all brain tumor segmentation tasks is a 70%, meanwhile the accuracy for such tasks which the paper said the model was already trained to do was >>90%. I know there is likely something I'm doing wrong so I was hoping you'd be able to help me figure it out, as the manual segmented data from BraTs differs from what biomedparse segments.

**What I am trying to do**

I am running zero-shot 3D segmentation on BraTS 2021 MRI volumes using the pretrained `biomedparse_v2.ckpt`. My goal is to evaluate how well the model segments glioma-related regions in BraTS, especially:

- whole tumor
- tumor core
- edema / non-enhancing FLAIR hyperintensity
- enhancing tumor

The BraTS labels I am using are:

- label 1 = necrotic / non-enhancing tumor core
- label 2 = edema / FLAIR abnormality
- label 4 = enhancing tumor

For evaluation I compute:

- WT = labels 1 + 2 + 4
- TC = labels 1 + 4
- ED = label 2
- ET = label 4
- NCR/NET = label 1

**Data and setup**

Dataset:
- BraTS 2021 TrainingSet
- HGG/GBM cohorts: `CPTAC-GBM`, `TCGA-GBM`, `UPENN-GBM`
- LGG cohort: `TCGA-LGG`

Model:
- BiomedParse v2
- pretrained checkpoint: `biomedparse_v2.ckpt`
- no fine-tuning

Inference:
- using `configs/model/biomedparse_3D`
- using `process_input(...)`, `postprocess(...)`, `merge_multiclass_masks(...)`, and `process_output(...)` as shown in the v2 README
- running on one MRI sequence at a time, usually FLAIR
- intensity preprocessing: clipping each volume to the 0.5th–99.5th percentile and rescaling to `[0, 255]`

Example inference logic:

```
imgs_tensor, pad_width, padded_size, valid_axis = process_input(img_u8, 512)
imgs_tensor = imgs_tensor.to(device).int()

input_tensor = {
    "image": imgs_tensor.unsqueeze(0),
    "text": [prompt],
}

with torch.no_grad():
    output = model(input_tensor, mode="eval", slice_batch_size=1)

mask_preds = output["predictions"]["pred_gmasks"]
mask_preds = F.interpolate(mask_preds, size=(512, 512), mode="bicubic", align_corners=False)

mask_preds = postprocess(mask_preds, output["predictions"]["object_existence"])
mask_preds = merge_multiclass_masks(mask_preds, [1])
mask_preds = process_output(mask_preds, pad_width, padded_size, valid_axis)
pred_mask = np.squeeze(mask_preds.detach().cpu().numpy()) > 0
```
